### DISCUSSION TITLE
Set default value as "true" "Remember me"

### DISCUSSION THREAD

**Discussion by jorozcof** [keycloak/keycloak#22193] [Q&A]
Is posible set default value as "true" "Remember me" ? When a user logs in and does not check the Remember Me option, is it possible to set the value to "true" by default?

![image](https://github.com/keycloak/keycloak/assets/115189644/a1a70ec8-8ed5-4303-bd21-47e07dddc66e)

**Comment by Sjoerd82** [comment]
Any update on this?

**Comment by darius-m** [comment]
Not sure if you can do it by setting an option in the admin console, but you can probably extend the default theme and add the `checked` [attribute to the input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox).

**Comment by mandree95** [comment]
Any update on this?

**Comment by CodingTil** [comment]
I have found a workaround for this - but Keycloak really needs to support this out-of-the-box, so we don't have to rely on such workarounds:
https://gist.github.com/CodingTil/8d1d98cc6bbdd6657bedfc0afadedc52

**Comment by mabartos** [comment]
@keycloak/core Any input on this?
